slurmstepd: error: Could not run slurm task_prolog [/u/slurm/bin/taskprolog]: Permission denied
[<81x1 sparse matrix of type '<type 'numpy.int64'>'
	with 1 stored elements in Compressed Sparse Row format>, <81x2 sparse matrix of type '<type 'numpy.int64'>'
	with 4 stored elements in COOrdinate format>, <81x1 sparse matrix of type '<type 'numpy.int64'>'
	with 1 stored elements in Compressed Sparse Row format>, <81x2 sparse matrix of type '<type 'numpy.int64'>'
	with 4 stored elements in COOrdinate format>, <81x5 sparse matrix of type '<type 'numpy.int64'>'
	with 32 stored elements in COOrdinate format>, <81x5 sparse matrix of type '<type 'numpy.int64'>'
	with 34 stored elements in COOrdinate format>, <81x1 sparse matrix of type '<type 'numpy.int64'>'
	with 1 stored elements in Compressed Sparse Row format>, <81x5 sparse matrix of type '<type 'numpy.int64'>'
	with 32 stored elements in COOrdinate format>, <81x8 sparse matrix of type '<type 'numpy.int64'>'
	with 93 stored elements in COOrdinate format>, <81x2 sparse matrix of type '<type 'numpy.int64'>'
	with 4 stored elements in COOrdinate format>, <81x5 sparse matrix of type '<type 'numpy.int64'>'
	with 34 stored elements in COOrdinate format>, <81x1 sparse matrix of type '<type 'numpy.int64'>'
	with 1 stored elements in Compressed Sparse Row format>, <81x2 sparse matrix of type '<type 'numpy.int64'>'
	with 4 stored elements in COOrdinate format>, <81x5 sparse matrix of type '<type 'numpy.int64'>'
	with 32 stored elements in COOrdinate format>, <81x5 sparse matrix of type '<type 'numpy.int64'>'
	with 34 stored elements in COOrdinate format>, <81x2 sparse matrix of type '<type 'numpy.int64'>'
	with 4 stored elements in COOrdinate format>, <81x1 sparse matrix of type '<type 'numpy.int64'>'
	with 1 stored elements in Compressed Sparse Row format>, <81x2 sparse matrix of type '<type 'numpy.int64'>'
	with 4 stored elements in COOrdinate format>, <81x1 sparse matrix of type '<type 'numpy.int64'>'
	with 1 stored elements in Compressed Sparse Row format>]
[4, 5, 7, 8, 9]
[(3, 5), (4, 5), (4, 5), (7, 8), (1, 2)]
aa and bb are
5
7
Multivariate Polynomial Ring in x_00, x_01, x_02, x_03, x_04, x_05, x_06, x_10, x_11, x_12, x_13, x_14, x_15, x_16, x_20, x_21, x_22, x_23, x_24, x_25, x_26, x_30, x_31, x_32, x_33, x_34, x_35, x_36, x_40, x_41, x_42, x_43, x_44, x_45, x_46 over Rational Field
([   1    0    0 x_00 x_01]
[   0    1    0 x_02 x_03]
[   0    0    1 x_04 x_05], [   1    0 x_10    0    0]
[   0    1 x_11    0    0]
[   0    0    0    1    0]
[   0    0    0    0    1], [0 1 0 0 0]
[0 0 1 0 0]
[0 0 0 1 0]
[0 0 0 0 1], [   1 x_30    0    0    0    0    0    0]
[   0    0    1    0    0    0    0    0]
[   0    0    0    1    0    0    0    0]
[   0    0    0    0    1    0    0    0]
[   0    0    0    0    0    1    0    0]
[   0    0    0    0    0    0    1    0]
[   0    0    0    0    0    0    0    1])
(81, 5)
[   1    0    0 x_00 x_01]
[   0    1    0 x_02 x_03]
[   0    0    1 x_04 x_05]
(81, 5)
[   1    0 x_10    0    0]
[   0    1 x_11    0    0]
[   0    0    0    1    0]
[   0    0    0    0    1]
(81, 5)
[0 1 0 0 0]
[0 0 1 0 0]
[0 0 0 1 0]
[0 0 0 0 1]
(81, 8)
[   1 x_30    0    0    0    0    0    0]
[   0    0    1    0    0    0    0    0]
[   0    0    0    1    0    0    0    0]
[   0    0    0    0    1    0    0    0]
[   0    0    0    0    0    1    0    0]
[   0    0    0    0    0    0    1    0]
[   0    0    0    0    0    0    0    1]
(81, 2)
Traceback (most recent call last):
  File "/data/scratch/kbari/BorderApolarity/BAsl3rk14m.py", line 239, in <module>
    AnnPlane1(H[int(sys.argv[1])])
  File "/data/scratch/kbari/BorderApolarity/BAsl3rk14m.py", line 146, in AnnPlane1
    print(g[j])
IndexError: tuple index out of range
